lib/prawn/document.rb

```diff
@@ -66,7 +66,8 @@ class Document
       :page_size, :page_layout, :margin, :left_margin,
       :right_margin, :top_margin, :bottom_margin, :skip_page_creation,
       :compress, :background, :info,
-      :text_formatter, :print_scaling
+      :text_formatter, :print_scaling,
+      :trailer, :enable_pdfa_1b
```

Comment:
I would rather make the ID deterministic so that we didn't have to make `trailer` accessible here.